src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -1968,6 +1968,17 @@ void MacroAssembler::ror_imm(Register dst, Register src, uint32_t shift, Registe
   orr(dst, dst, tmp);
 }
 
+// rotate left with shift bits, 32-bit version
+void MacroAssembler::rol32_imm(Register dst, Register src, uint32_t shift, Register tmp) {
```

Comment:
You could actually name it `rolw_imm` so that you can easily match it to https://github.com/riscv/riscv-bitmanip/blob/6d33919c661a838f6ae89a05650b3ee6aa303440/bitmanip/insns/rolw.adoc#L3 when Zbb is available.

Reply:
OK.